slangpy/reflection/reflectiontypes.py

```
@@ -1761,6 +1763,24 @@ def can_convert_to_int(value: Any):
 
 
+def can_convert_to_bool(value: Any):
+    if isinstance(value, bool):
+        return True
+    if not isinstance(value, str):
+        return False
+    normalized = value.strip().lower()
+    return normalized in ("true", "false", "bool(1)", "bool(0)")
+
+
+def convert_bool_to_int(value: Any):
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)